sig-ui/annual-report-2021.md

```diff
@@ -0,0 +1,102 @@
+# SIG UI 2021 Annual Report
+
+**Authors:**
+
+Sebastian Floreks ([@floreks](https://github.com/floreks))
```

Comment:
s/Floreks/Florek

Reply:
:man_facepalming: Putting the S at the end is muscle memory